web/docs/auth/overview.md

```diff
@@ -504,8 +504,382 @@ try {
 </TabItem>
 </Tabs>
 
+## Customizing the Signup Process
+
+Sometimes you want to include **extra fields** in your signup process, like first name and last name.
+
+In Wasp, in this case:
+
+- You need to define the new backend fields
+- You need to customize the `SignupForm`
+
+Other times, you might need to add some **extra UI** elements to the form, like a checkbox for terms of service. In this case, customizing only the UI components is enough.
```

Comment:
> Avoid language that invalidate struggle, such as "easy", "just", "obviously", etc. For reference, see [Words To Avoid in Educational Writing](https://css-tricks.com/words-avoid-educational-writing/).

https://github.com/vuejs/v2.vuejs.org/blob/master/writing-guide.md

Reply:
Sure, that makes sense, but this wasn't for the purpose of making it sound easy, the purpose was to make a contrast compared to the previous part where we had to do two things, while here we do only one thing. When I was reading it the first time, I thought that is not very clear, that we are doing only half of the job here.
I get what they mean with not using "just", but I don't think this case falls under that. Certainly not all usages of "just" should be eliminated, just those that invalidate struggle, like "now let's just implement this thing" or "the only thing left is just running the command".
Maybe there is another way to indicate this is only part of what was needed above it?

Reply:
My bad, I jumped right at the word without understanding that real meaning behind the change. The change makes a lot of sense 👍